packages/assistant-stream/src/core/json/ObjectStreamAccumulator.ts

```
@@ -0,0 +1,76 @@
+        );
+      case "append-text":
+        return ObjectStreamAccumulator.updatePath(state, op.path, (current) => {
+          if (current === undefined) return op.value;
+
+          if (typeof current !== "string")
+            throw new Error(`Expected string at path [${op.path.join(", ")}]`);
+          return current + op.value;
+        });
+
+      default:
+        const _exhaustiveCheck: never = type;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)